scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/SignatureAlgorithmsExtension.java

```diff
@@ -0,0 +1,111 @@
+package org.eclipse.californium.scandium.dtls;
```

Comment:
For new file we use a [license header](https://github.com/eclipse/californium/blob/master/license_header_template.txt) 

`(company) name` is for the company, if this contribution was developed on a job for that company, or your name, if you developed it in your free time.

Reply:
Hi. :) I have added the license header.